practice.js

```diff
@@ -169,6 +169,21 @@ document.getElementById("levelSelector").addEventListener("click", (e) => {
     }
 })
 
+
+document.addEventListener('keypress', (e)=>{
```

Comment:
Add a paragraph on `practice.html` below `start button` like `Press Space key to start practicing`.